src/asserts/big-number-equal-to-assert.js

```diff
@@ -26,23 +29,23 @@ export default function bigNumberEqualToAssert(value) {
     throw new Error('A value is required.');
   }
 
-  this.value = new BigNumber(value);
+  this.value = createBigNumber(value);
 
   /**
    * Validation algorithm.
    */
 
   this.validate = value => {
     try {
-      const number = new BigNumber(value);
+      const number = createBigNumber(value);
```

Comment:
We shouldn't need to extract repeated logic into one function to be imported everywhere, since we already have an assert that does all that work for us.

So we should be able to import the `BigNumber` assert:

```js
import BigNumberAssert from './big-number-assert';

const is = Assert.extend({
  BigNumber: BigNumberAssert
});
```

And then here use that assert directly instead of extracted functions:

```js
is.bigNumber({ validateSignificantDigits }).validate(value);

try {
  const number = new BigNumber(value);
```

WDYT?

Reply:
I like it.
I've update the PR.